build/patches/Remove-http-referrals-in-cross-origin-navigation.patch

```diff
@@ -0,0 +1,103 @@
+From: uazo <uazo@users.noreply.github.com>
+Date: Wed, 21 Sep 2022 12:28:17 +0000
+Subject: Remove http referrals in cross origin navigation
+
+and adds a flag to disable them completely
+---
+ chrome/android/java/res/xml/privacy_preferences.xml       | 4 ++++
+ .../chrome/browser/privacy/settings/PrivacySettings.java  | 8 ++++++++
+ .../browser/ui/android/strings/android_chrome_strings.grd | 3 +++
+ content/browser/renderer_host/navigation_request.cc       | 7 +++++++
+ services/network/public/cpp/resource_request.h            | 2 +-
+ 5 files changed, 23 insertions(+), 1 deletion(-)
+
+diff --git a/chrome/android/java/res/xml/privacy_preferences.xml b/chrome/android/java/res/xml/privacy_preferences.xml
+--- a/chrome/android/java/res/xml/privacy_preferences.xml
++++ b/chrome/android/java/res/xml/privacy_preferences.xml
+@@ -57,6 +57,10 @@
+         android:title="@string/close_tabs_on_exit_title"
+         android:summary="@string/close_tabs_on_exit_summary"
+         android:defaultValue="false" />
++    <org.chromium.components.browser_ui.settings.ChromeSwitchPreference
++        android:key="enable_referrers"
++        android:title="@string/enable_referrers_title"
++        android:defaultValue="false" />
```

Comment:
Shouldn't the default be `true`?

Reply:
it's the same. the value is set from see line 54 while the default is that of prefs